SCons/Tool/ninja/NinjaState.py

```diff
@@ -536,6 +536,7 @@ def __init__(self, env):
             # TODO: use command action #3573
             "installFunc": _install_action_function,
             "MkdirFunc": _mkdir_action_function,
+            "Mkdir": _mkdir_action_function,
```

Comment:
while investing the warning message about a directory already existing I dug into ninja's conversion of Mkdir function, and found that the alternate use via Command(action='Mkdir') was not supported so I added it.